cerberus/cerberus.py

```diff
@@ -298,6 +316,9 @@ def _validate_definition(self, definition, field, value):
             if validator:
                 validator(definition[rule], field, value)
 
+        if 'rename' in definition:
```

Comment:
Makes sense, I'll put the rename first. As for adding a `remove_unknown` rule to the schema, will do.